pkg/apis/core/v1alpha1/types.go

```diff
@@ -204,11 +206,11 @@ type CDIPhase string
 
 // CDIStatus defines the status of the CDI installation
 type CDIStatus struct {
-	Phase           CDIPhase       `json:"phase,omitempty"`
-	Conditions      []CDICondition `json:"conditions,omitempty" optional:"true"`
```

Comment:
Changing the type of a field is something to avoid when possible.  The right thing to do when necessary is to bump the version of the CRD.  Are we not at all concerned about instances of this CR in the wild?

Also, Kubevirt did not migrate to this new Conditions struct/package.  Why are we? `CDICondition` also pretty much matches `KubeVirtCondition`.

Reply:
I pulled all of it from here: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/docs/conditions.md Not sure why KubeVirt decided not to follow that.

Reply:
As for the type, https://github.com/openshift/custom-resource-status/blob/master/conditions/README.md describes what it should be.

Reply:
If we have to switch condition types it may be worth the exercise of bumping the CRD version.  I'm wondering what would happen if the operator is updated when old CR exists.  At a minimum, the `LastProbeTime` of `CDICondition` will be lost which is probably not the end of the world but I'm not sure what else would happen.